src/test/kotlin/com/pubnub/contract/member/step/WhenSteps.kt

```diff
@@ -14,21 +14,20 @@ class WhenSteps(
     @When("I get the channel members")
     fun i_get_the_channel_members() {
         world.pubnub.getChannelMembers(channel = memberState.channelId()).sync()?.let {
-            memberState.members = it.data
+            memberState.returnedMembers = it.data
             world.responseStatus = it.status
         }
     }
 
     @When("I set a channel member including custom and UUID with custom")
     fun i_set_a_channel_member_including_custom_and_uuid_with_custom() {
-        val memberInput = PNMember.Partial(uuidId = memberState.member.uuid?.id!!)
         world.pubnub.setChannelMembers(
             channel = memberState.channelId(),
             includeCustom = true,
             includeUUIDDetails = PNUUIDDetailsLevel.UUID_WITH_CUSTOM,
-            uuids = listOf(memberInput)
+            uuids = memberState.members.map { PNMember.Partial(uuidId = it.uuid?.id!!) }
```

Comment:
It's your call but I would assign 
memberState.members.map { PNMember.Partial(uuidId = it.uuid?.id!!) }
to variable(e.g. myVariable) and then 
uuids = myVariable